skiko/build.gradle.kts

```diff
@@ -186,6 +186,7 @@ val linkWasm = tasks.register<LinkSkikoWasmTask>("linkWasm") {
         "-l", "GL",
         "-s", "USE_WEBGL2=1",
         "-s", "OFFSCREEN_FRAMEBUFFER=1",
+        "-s", "ALLOW_MEMORY_GROWTH=1", // Is there a better way? Should we use `-s INITIAL_MEMORY=X`?
```

Comment:
I suggest to add TODO because it's easier to find all places where we actually consider to improve/investigate something

Reply:
Unfortunate part here is that per https://groups.google.com/g/emscripten-discuss/c/m10q3iDixnQ this option is incompatible with threading. Would be cool to come up with some sensible approach.

Reply:
But after some thinking, maybe that's OK, as potentially Skia can manipulate with rather large arrays, so we have to allow it to grow memory, and maybe we can do concurrrency on JS side (i.e. Workers) if needed.

Reply:
added TODO to not forget about this discussion.